.husky/pre-commit

```diff
@@ -1,17 +0,0 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-echo "🚀   > Pretty quick reformat..."
-npx pretty-quick --staged
-
-echo "🚀   > JSdoc regen..."
-npx jsdoc src-electron src-shared -r -d ./generated-html/
-
-echo "🚀   > Eslint..."
-npx eslint --ext .js,.vue src src-electron src-shared src-script test
-
-echo "🚀   > Reset version in package.json to 0.0.0..."
-src-script/zap-update-package-version.js -fake
-
-echo "🚀   > Check versions in package-lock.json..."
-src-script/zap-validate-metafiles.js
```

Comment:
Why is this removed?

Reply:
Wanted to quickly push draft PR, reverted this once the draft was finalized